k9mail/src/main/java/com/fsck/k9/activity/compose/RecipientPresenter.java

```diff
@@ -370,6 +370,10 @@ public ComposeCryptoStatus getCurrentCryptoStatus() {
     }
 
     public boolean isForceTextMessageFormat() {
+        if (!cryptoEnablePgpInline) {
+            return false;
+        }
```

Comment:
This is a bit double negativey

Wouldn't

    if (cryptoEnablePgpInline) {
       ComposeCryptoStatus ....
    } else {
       return false;
    }

be more readable.